ee_tests/src/page_objects/user_profile.page.ts

```diff
@@ -60,26 +60,34 @@ export class CleanupUserEnvPage extends AppPage {
     support.debug('... checking if erase button is there - OK');
   }
 
+  /* An intermittent error is resulting in some user enviroment resets failing
+       https://github.com/openshiftio/openshift.io/issues/1637
+       If this happens - try one more time - if that fails something serious
+       has happened and the test will/and should fail */
   async cleanup(username: string) {
     await this.cleanupSupport(username);
 
     await browser.wait(until.presenceOf(this.alertBox), support.LONG_WAIT);
     let alertText = await this.alertBox.getText();
 
-    /* An intermittent error is resulting in some user enviroment resets failing
-       https://github.com/openshiftio/openshift.io/issues/1637
-       If this happens - try one more time - if that fails something serious
-       has happened and the test will/and should fail */
     support.info('Alert text: ' + alertText);
```

Comment:
Could we refactor function `cleanup()` in such a way that all repeating code will go to `cleanupSupport()`?